framework/db/sqlite/QueryBuilder.php

```diff
@@ -141,16 +141,13 @@ public function resetSequence($tableName, $value = null)
                 $key = reset($table->primaryKey);
                 $tableName = $db->quoteTableName($tableName);
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX({$key}) FROM {$tableName}")->queryScalar();
```

Comment:
I agree about quoting but disagree about `{$}` - I think it’s looks clear that it will be evaluated as a variable and won't be affected by string changes at all.